Dockerfile

```diff
@@ -9,29 +9,40 @@ LABEL name="openshift-wetty-client" \
       io.k8s.description="Provides oc cli tooling in a web browser" \
       io.openshift.tags="openshift,wetty,oc,ubi7"
 
+ARG OC_MAJOR_VERSION
+ARG OC_VERSION
+ARG ODO_VERSION
+
 ENV NODEJS_VERSION=10 \
     NODE_ENV=production \
     NPM_CONFIG_PREFIX=$HOME/.npm-global \
     PATH=$HOME/node_modules/.bin/:$HOME/.npm-global/bin/:$PATH \
     WETTY_NUMBER_OF_USERS=60 \
     WETTY_USERNAME_PREFIX=user \
-    WETTY_PASSWORD_PREFIX=password
+    WETTY_PASSWORD_PREFIX=password \
+    OC_MAJOR_VERSION=${OC_MAJOR_VERSION:-3} \
```

Comment:
should this be `-3` or `3`

Reply:
I just pushed a new change to the branch that fixes the typo with the }. Not sure why my docker build didn't fail there. It should be a -3. That is the syntax for a Variable or default in bash.

Reply:
hmm, not seeing the change yet - when you push it to `dmc5179:ocversion` it will show here too

Reply:
oops, it was hanging for my password. For some reason I checked it out with https....haha